Mathlib/Analysis/InnerProductSpace/Calculus.lean

```diff
@@ -245,6 +245,38 @@ theorem DifferentiableAt.norm (hf : DifferentiableAt ℝ f x) (h0 : f x ≠ 0) :
   ((contDiffAt_norm 𝕜 h0).differentiableAt le_rfl).comp x hf
 #align differentiable_at.norm DifferentiableAt.norm
 
+theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (fun x : ℝ ↦ abs x) 0 := by
+  rw [DifferentiableAt]
+  push_neg
+  intro f
+  simp only [HasFDerivAt, HasFDerivAtFilter, abs_zero, sub_zero,
+    Asymptotics.isLittleO_iff, norm_eq_abs, not_forall, not_eventually, not_le, exists_prop]
+  use 2⁻¹
+  use by norm_num
```

Comment:
```suggestion
  use 2⁻¹, by norm_num
```
you can combine use steps like this

Reply:
ah I see that you already know this ;) as you use it below (no pun intended) still i think it makes the proof cleaner to put them together

Reply:
Haha. Yes, I should remember to combine the `use`s.